### ISSUE TITLE
Open additional menu links in the same tab

### ISSUE THREAD

**Issue by zablvit** [jaegertracing/jaeger-ui#275]
### tl;dr
Possibility to open [additional menu links in global nav](https://www.jaegertracing.io/docs/1.8/deployment/#UIConfiguration) in the same tab (`target="_self"`)

### Detailed use case

I'm using [keycloak gatekeeper](https://github.com/keycloak/keycloak-gatekeeper) to add login page to jaeger query and I want to add "Logout" link in top nav which will point to `/oauth/logout` which will end keycloack session and redirect to login page.

In current state this will open new tab, perform logout and redirect to login screen, however this all happens in a new tab and jaeger ui tab is still open. I need this all to happen in the same tab. 

## Problem - what in Jaeger blocks you from solving the requirement?
Hardcoded `target="_blank"` in [here](https://github.com/jaegertracing/jaeger-ui/blob/feeaccd1502b2748b96ca3619e9c3dcb02bc0ed3/packages/jaeger-ui/src/components/App/TopNav.js#L62)

## Proposal - what do you suggest to solve the problem or improve the existing situation?

Add `openInSameTab` boolean field to [ConfigMenuItem type](https://github.com/jaegertracing/jaeger-ui/blob/feeaccd1502b2748b96ca3619e9c3dcb02bc0ed3/packages/jaeger-ui/src/types/config.js#L17) which, if set to true will set link target to ___self__
